src/main/java/org/jabref/gui/BasePanel.java

```diff
@@ -808,6 +811,33 @@ private void paste() {
         }
     }
 
+    private void copyTitle() {
+        List<BibEntry> bes = mainTable.getSelectedEntries();
+        if (!bes.isEmpty()) {
+            storeCurrentEdit();
+            List<String> titles = new ArrayList<>(bes.size());
+
+            // Collect all non-null keys.
+            for (BibEntry be : bes) {
+                be.getTitle().ifPresent(titles::add);
```

Comment:
I couldn't find the helper to transform an optional into a stream in the [Optional](https://docs.oracle.com/javase/8/docs/api/java/util/Optional.html) documentation. 
I believe this is something that was added later in JDK 9.
I looked into [other ways](http://stackoverflow.com/questions/22725537/using-java-8s-optional-with-streamflatmap) to use flatMap, but these did not look very nice to me.

Do you think I should change it any way or just leave it like it is now (with the filter + map)?